roles/lib_utils/src/generate.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+'''
+  Generate the openshift-ansible/roles/lib_openshift_cli/library/ modules.
+'''
+
+import os
+
+# pylint: disable=anomalous-backslash-in-string
+GEN_STR = "#!/usr/bin/env python\n"                                  + \
+          "# pylint: disable=missing-docstring\n"                    + \
+          "#     ___ ___ _  _ ___ ___    _ _____ ___ ___\n"          + \
+          "#    / __| __| \| | __| _ \  /_\_   _| __|   \\\n"        + \
+          "#   | (_ | _|| .` | _||   / / _ \| | | _|| |) |\n"        + \
+          "#    \___|___|_|\_|___|_|_\/_/_\_\_|_|___|___/_ _____\n"  + \
+          "#   |   \ / _ \  | \| |/ _ \_   _| | __|   \_ _|_   _|\n" + \
+          "#   | |) | (_) | | .` | (_) || |   | _|| |) | |  | |\n"   + \
+          "#   |___/ \___/  |_|\_|\___/ |_|   |___|___/___| |_|\n"
+
+OPENSHIFT_ANSIBLE_PATH = os.path.dirname(os.path.realpath(__file__))
+
+FILES = {'yedit.py':  ['doc/license', 'doc/yedit', 'class/base.py', 'class/yedit.py', 'ansible/yedit.py'],
```

Comment:
@tbielawa,

Thanks for the feedback.  I'm fine with moving the generated files into a yaml file.  I'm a fan of auto-generating but in this case I'm not because of a couple of reasons.  

1.   I'd like for everyone who comes in to understand the order in which the files are generated and attempt to be transparent about what is happening.

1. I want it to be flexible.  If there are certain libraries that require certain sources, I don't want all files getting them.  Thus being explicit is what I'd prefer. 

The overall suggestion of putting them into a yaml file is fine.  I put them into a dictionary structure so ultimately putting them into a yaml is better for readability and keeps the dictionary format.